### PR TITLE
Small cleanups

### PR DESCRIPTION
**Pull request by andre-rosa**
Separating few cleanups from https://github.com/ros-infrastructure/superflore/pull/168 .

### CODE REVIEW

File: superflore/utils.py [hunk 1]
```diff
@@ -202,7 +199,7 @@ def get_license(l):
     elif re.search(cc_re, l, f):
         return 'CC-BY-SA-3.0'
     elif re.search(boost_re, l, f):
-        return 'Boost-1.0'
```
Comment (allenh1) [inline superflore/utils.py]: Please revert this change

Comment (allenh1) [reply superflore/utils.py]: This still does not work with Gentoo, which gets very upset if this is not matched exactly.

File: superflore/generators/bitbake/yocto_recipe.py [hunk 2]
```diff
@@ -58,17 +58,14 @@ def __init__(
...
-                self.downloadArchive()
-                md5_cache[self.getArchiveName()] = hashlib.md5(
-                    open(self.getArchiveName(), 'rb').read()).hexdigest()
-                sha256_cache[self.getArchiveName()] = hashlib.sha256(
-                    open(self.getArchiveName(), 'rb').read()).hexdigest()
+            self.downloadArchive()
+            md5_cache[self.getArchiveName()] = hashlib.md5(
+              open(self.getArchiveName(), 'rb').read()).hexdigest()
+            sha256_cache[self.getArchiveName()] = hashlib.sha256(
+              open(self.getArchiveName(), 'rb').read()).hexdigest()
```
Comment (allenh1) [inline superflore/generators/bitbake/yocto_recipe.py]: I think the linter is unhappy with this whitespace change.

File: superflore/generators/bitbake/run.py [hunk 3]
```diff
@@ -31,38 +32,29 @@
...
 from superflore.utils import warn
 
-# Modify if a new distro is added
-active_distros = ['indigo', 'kinetic', 'lunar']
-
 
 def main():
-    preserve_existing = True
     overlay = None
+    preserve_existing = True
```
Comment (allenh1) [inline superflore/generators/bitbake/run.py]: to reduce the delta, does the location of this variable need to change?

File: superflore/generators/bitbake/gen_packages.py [hunk 4]
```diff
@@ -71,10 +71,10 @@ def regenerate_installer(
             for f in glob.glob('%s*.patch' % patch_path)
         ]
     if preserve_existing and existing:
-        ok("recipe for package '%s' up to date, skpping..." % pkg)
+        ok("recipe for package '%s' up to date, skipping..." % pkg)
```
Comment (allenh1) [inline superflore/generators/bitbake/gen_packages.py]: *facepalm* and to think, I almozt niver hab tipos

File: .vscode/launch.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
+{
```
Comment (allenh1) [inline .vscode/launch.json]: I'm not sure how I feel about the `.vscode` files being added, to be frank. I don't hate this in any sense, but, as I don't use this editor (if you can't tell from my atrocious spelling abilities), I'm not sure I want to be responsible for maintaining this.

Is this something that should absolutely be included in the repository, or could you simply hold on to these files?

@tfoote do you have any opinions on this? I'm not just not sure this is best practice.

File: superflore/utils.py (hunk 1 above)
Comment (andre-rosa) [reply superflore/utils.py]: Done; bitbake is also happy about 'Boost-1.0'.

File: superflore/generators/bitbake/run.py (hunk 3 above)
Comment (andre-rosa) [reply superflore/generators/bitbake/run.py]: Right, otherwise it's a small one line delta.

File: .vscode/launch.json (hunk 5 above)
Comment (andre-rosa) [reply .vscode/launch.json]: I can definitely remove it. I thought that would be helpful for other people and non-intrusive but it's totally fine to carry it myself.

Comment (tfoote) [reply .vscode/launch.json]: Yeah, in general we try to avoid development tool specific as it's both a personal choice of what editor to use as well as how you want to set it up.

Comment (andre-rosa) [reply .vscode/launch.json]: Thanks, I've dropped the .vscode files.

**Comment by allenh1** [comment]
lgtm -- waiting for @tfoote to review and we should be ready to merge this one.

Thanks!

**Comment by andre-rosa** [comment]
Thank you _both_!
